src/clients/mobiles/tabs/index.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import { SocketService } from "../utils/socket";
+import { connect, Dispatch } from "react-redux";
+import { State, Dispatcher, mapDispatchToProps, mapStateToProps, ReduxStore, ReduxState } from "../store/index";
+import { OPEN_NEW_TAB, REMOVE_TAB, REFRESH_TAB } from "../../../communication/actions";
+
+export interface TabsProps {
+    socketService: SocketService
+}
+
+class TabsContainer extends React.Component<TabsProps & ReduxStore, {}>  {
+
+    public newTab() {
+        this.props.socketService.sendTabsActionsMessageFromClient({
+            actionType: OPEN_NEW_TAB,
+            action: 'https://www.dashlane.com'
```

Comment:
🤨

Reply:
Yeah not really generic for now ^^^
But the point was to have something working